src/shapesdemo/KeylessShapeTypeObject.cxx

```diff
@@ -41,9 +41,9 @@ using namespace eprosima::fastrtps::rtps;
 void registerKeylessShapeTypes()
 {
     TypeObjectFactory *factory = TypeObjectFactory::get_instance();
-    factory->add_type_object("shapes_demo_typesupport::idl::KeylessShapeType", shapes_demo_typesupport::idl::GetKeylessShapeTypeIdentifier(true),
+    factory->add_type_object("shapes_demo_typesupport::idl::dds_::KeylessShapeType_", shapes_demo_typesupport::idl::GetKeylessShapeTypeIdentifier(true),
```

Comment:
Have you opened an internal task (Redmine) to track this issue?

Reply:
It has been created